app/config/locale/translations/en.json

```diff
@@ -4,13 +4,13 @@
     "settings.direction": "ltr",
     "emails.sender": "%s Team",
     "emails.verification.subject": "Account Verification",
-    "emails.verification.hello": "Hey {{user}}",
+    "emails.verification.hello": "Hello {{user}}",
```

Comment:
If there is no user and you set user to `,`, then you still get `Hello ,` with unwanted space